### DISCUSSION TITLE
Apple support

### DISCUSSION THREAD

**Discussion by amerkoleci** [amerkoleci/alimer#19] [Ideas]
Hi,
I don't have apple PC or iOS hardware, if anyone can provide help or macchine support I will gladly add apple with metal graphics backend support.

Ping me in case you want to help,

Thanks!

**Comment by IceReaper** [comment]
I have an apple M1 mac mini solely for the purpose of testing my code prototypes.
I would be open for remote access while im overseeing it, if that would help.
